tests/explicit-nulls/neg/from-nullable.scala

```diff
@@ -0,0 +1,6 @@
+import scala.annotation.experimental
+
+@experimental def testFromNullable =
+  val s: String | Null = "abc"
+  val sopt1: Option[String] = Option(s) // error
+  val sopt2: Option[String] = Option.fromNullable(s) // ok
```

Comment:
Please also add a run test, because currently the body of `fromNullable` could be `???` and the CI would pass.

Reply:
Added